.github/workflows/action-test.yml

```diff
@@ -21,6 +21,7 @@ on:
   push:
     branches:
     - main
+    - compare_datetimes
```

Comment:
Remove this?

Reply:
on it right after the last OSX test passes